src/common/components/nav/nav.bundle.js

```diff
@@ -7,11 +7,11 @@
   // angular.js to load itself into
   const angular = (window.angular = {});
 
-  require('./nav.component');
+  const nav = require('./nav.component');
```

Comment:
Will it include the nav component in the bundle?  Or just assigned it to the constant?